validation/standard/standard-versions.toml

```diff
@@ -0,0 +1,68 @@
+["op-contracts/v1.4.0-rc.3"]
```

Comment:
From discussion with @mds1  today:

We could simplify this by keeping only` 1.3.0` and `1.4.0`.

OP Mainnet and OP Sepolia should both be `1.4.0`.

Metal, Mode and Zora (both mainnet and sepolia) should be `1.3.0`. 

Base Sepolia should also be `1.3.0`.

Reply:
The test matches against either 1.3.0 or 1.4.0, and this results in: 

<img width="493" alt="Screenshot 2024-06-20 at 21 15 59" src="https://github.com/ethereum-optimism/superchain-registry/assets/1833419/09117d5c-045a-4020-8a12-f4f534aade08">